src/core/src/test/java/org/apache/jmeter/engine/DistributedRunnerTest.java

```diff
@@ -188,4 +188,4 @@ public void setHost(String host) {
             this.host = host;
         }
     }
-}
+}
```

Comment:
Please keep the final newline: https://github.com/apache/jmeter/blob/dc91e8bcf6bcb6ec6572672b12ec9fbb389c7159/.editorconfig#L5

Reply:
Thank you! Done.